core/client-internal/src/test/java/tachyon/client/file/BaseFileSystemTest.java

```diff
@@ -69,7 +69,6 @@
    */
   @Before
   public void before() {
-    ClientContext.reset();
```

Comment:
o, but it seems now in `ClientContext`, we don't reset the subsystems' context any more, right?